app/com/linkedin/drelephant/schedulers/Scheduler.java

```diff
@@ -99,6 +99,20 @@
    */
   public String getJobName();
 
+  /**
+   * Return the name of the org
+   *
+   * @return the org name
+   */
+  public String getOrganization();
```

Comment:
Can this be changed to an abstract class with final implementations for og and sub org so that its available for all schedulers.